Healthy/Classes/Modules/Login/LoginViewController.swift

```diff
@@ -42,7 +42,16 @@ final class LoginViewController: UIViewController {
 
 private extension LoginViewController {
     @IBAction func didTapSignIn(_ sender: Any) {}
+
     @IBAction func didTapSignUp(_ sender: Any) {}
+
+    @IBAction func didTapSignInWithGoogle(_ sender: Any) {
```

Comment:
Please create a wrapper for this. Any 3rd party dependency should never by added to the code without an adapter.